src/resolver/article-table.ts

```diff
@@ -0,0 +1,44 @@
+import { MongoDBTableBase } from "../mongodb/mongodb-manager";
+import { Article } from "../../common/models";
+import * as _ from "lodash";
+
+export class ArticleTable extends MongoDBTableBase("articles") {
+    public addArticle(article: Article) {
```

Comment:
Some comments to the model: 
author -> authors
status -> can it be a enum? or maybe a boolean called isPublished, whichever
pusblishTime -> better be stored as integer. unix timestamp

Some fields I would like:
  slug: string
  preview: string (or excerpt or description or any other name)
Also create time and last modified time should be still useful. both should be in unix timestamp

Reply:
just see the type of status is ArticleStatus. Please use that type for graphql as well!